scripts/ingests/ingest_bard14_spectra.py

```diff
@@ -0,0 +1,164 @@
+from scripts.ingests.ingest_utils import *
+from scripts.ingests.utils import *
+from astropy.table import Table
+from astropy.io import ascii
+import astropy.units as u
+from astropy.coordinates import Angle
+   
+
+SAVE_DB = False  # True: save the data files(json) in addition to modifying the .db file
+RECREATE_DB = True  # recreates the .db file from the data files
+# LOAD THE DATABASE
+db = load_simpledb('SIMPLE.db', recreatedb=RECREATE_DB)
+
+#bard14 url : https://docs.google.com/spreadsheets/d/11o5NRGA7jSbHKaTNK7SJnu_DTECjsyZ6rY3rcznYsJk/edit#gid=0
+
+SHEET_ID = '11o5NRGA7jSbHKaTNK7SJnu_DTECjsyZ6rY3rcznYsJk'
+SHEET_NAME = 'bard14'
+full = 'all'
+
+url = f'https://docs.google.com/spreadsheets/d/{SHEET_ID}/gviz/tq?tqx=out:csv&sheet={full}'
+
+bard14_table = ascii.read(
+    url,
+    format="csv",
+    data_start=1,
+    header_start=0,
+    guess=False,
+    fast_reader=False, 
+    delimiter=",",
+)
+
+#print result table
+print(bard14_table.info)
+
+
+#Loop through data and update spectra
+def updating_spectra(db):
+    for row in bard14_table[0:19]:  
+
+        # Print spectra information
+        print("Spectra Information:")
+        
+        for col_name in row.colnames:
+            print(f"{col_name}: {row[col_name]}")
+
+#Call spectra function
+updating_spectra(db)
+
+#Ingest spectra loop 2
+            
+def update_all_spectra(db):
+    updating_spectra(db)
+
+    with db.engine.begin() as conn:
+        for entry in update_all_spectra:
+            source_value = entry['Source']
+            spectrum_value = entry['Spectrum']
+            original_spectrum_value = entry['Original Spectrum']
+            regime_value = entry["regime"]
+            telescope_value = entry["telescope"]
+            instrument_value = entry["instrument"]
+            mode_value  = entry["mode"]
+            observation_date_value = entry["observation date"]
+            spectrum_comments_value = entry["spectrum comments"]
+            spectrum_reference_value = entry["spectrum reference"]
+            ra_value = entry["ra"]
+            dec_value = entry["dec"]
+            aperture_value = entry["aperture"]
```

Comment:
The only fields that need updating are the spectrum and original_spectrum so you don't need any of these other fields.
```suggestion
   
```